lib/primer/view_components/linters/argument_mappers/button.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require_relative "conversion_error"
+require_relative "system_arguments"
+
+module ERBLint
+  module Linters
+    module ArgumentMappers
+      # Maps classes in a button element to arguments for the Button component.
+      class Button
+        SCHEME_MAPPINGS = {
```

Comment:
Do you think we could reuse the scheme mappings from the ButtonComponent? (I can also see why we'd rather not include the component for dependency reasons)

My concern would be about keeping these in sync.

Reply:
the problem is everything that we need to load to get access to those constants :/